pallets/ocex/src/lib.rs

```diff
@@ -395,9 +399,21 @@ pub mod pallet {
 		) -> DispatchResult {
 			let user = ensure_signed(origin)?;
 			// TODO: Check if asset is enabled for deposit
-			Self::transfer_asset(&user, &Self::get_custodian_account(), amount, asset)?;
+
+			ensure!(amount.saturated_into::<u128>() < DEPOSIT_MAX, Error::<T>::DepositOverflow);
```

Comment:
If this is `DEPOSIT_MAX` shouldn't be checked including this value? e.g.  `amount.saturated_into::<u128>() <= DEPOSIT_MAX`? because maximum is still included value from semantic point of view.

Reply:
Updated ✅